### DISCUSSION TITLE
battlefield 2042 is new version after downloading the old version with steam command download depot

### DISCUSSION THREAD

**Discussion by JL-victor** [mmvanheusden/SteamDepotDownloaderGUI#133] [Q&A] [answered]
cliack on the launcher to start with the old version,but when i load the ea app,it becomes the new version
![屏幕截图 2023-10-25 203715](https://github.com/mmvanheusden/SteamDepotDownloaderGUI/assets/148459960/5caa93d1-4689-453e-aa14-b70fbc3d8334)
![屏幕截图 2023-10-25 204013](https://github.com/mmvanheusden/SteamDepotDownloaderGUI/assets/148459960/843d49fb-8b8a-405f-b712-b632bd969bf7)

**Comment by mmvanheusden** [comment, marked as answer]
Yeah it happens with some games, the trick here is to replace your downloades files with the original files.

you go to the cyberpunk folder, and rename it to something different, and then copy and paste the downloaded files to the original folder
